ansible/roles/ocp4-workload-integreatly/defaults/main.yml

```diff
@@ -24,4 +24,27 @@ silent: False
 ocp4_workload_integreatly_defaults:
     variable_1: "value 1"
     variable_2: "value 2"
-    variable_secret: ""
\ No newline at end of file
+    variable_secret: ""
+
+action: install
```

Comment:
Can we put these variables into the workload dictionary? I'd really like to avoid top level variables as much as possible to avoid clashes.

Also: should this be a separate workload just like Minio?

Reply:
And use users generated by the ocp4-workload-authentication workload?

Reply:
@wkulhanek thanks for the feed back it is appreciated 
- sure I can put the vars into the workload dictionary :+1: 
- Not sure a separate workload make sense it doesn't have any usability outside of a post install task for Integreatly
- No the users are 3scale tenant admins , not ocp4 htdpasswd users